### PR TITLE
Fix #90: Initial work for kapitan init

### LINKED ISSUE #90: kapitan init
`kapitan init` command to create an empty/example project skeleton.

### PR DESCRIPTION
**Pull request by uberspot**
Fixes issue #90 

## Proposed Changes

  - Adds kapitan init as a new command. Recommendations on what further dirs/files should be created are welcome. :+1:

### CODE REVIEW

File: kapitan/initialiser.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    """ Initialises a directory with a recommended skeleton structure
+    Args:
+        init_directory (string): path which to initialise, directory is assumed to exist
+    """
+    if not os.path.isdir(init_directory):
+        logger.info("Provided path is not a directory, or doesn't exist. Exiting...")
+        sys.exit(1)
+
+    dir_structure = [
+            "compiled",
```
Comment (adrianchifor) [inline kapitan/initialiser.py]: compiled is generated on compile

File: kapitan/cli.py [hunk 2]
```diff
@@ -210,6 +211,11 @@ def main():
                              default=from_dot_kapitan('lint', 'inventory-path', './inventory'),
                              help='set inventory path, default is "./inventory"')
 
+    init_parser = subparser.add_parser('init', help='initialize a directory with the recommended kapitan project skeleton.')
+    init_parser.add_argument('--init-directory',
```
Comment (adrianchifor) [inline kapitan/cli.py]: maybe --dir

File: kapitan/initialiser.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        directory (string): path which to initialise, directory is assumed to exist
+    """
+    
+    templates_directory = os.path.join(os.path.dirname(
+        __file__), 'inputs', 'templates')
+
+    copy_tree(templates_directory, directory)
+
+    # create lib/kube.libjsonnet
+    kube_jsonnet_lib_path = os.path.join(os.path.dirname(
```
Comment (ramaro) [inline kapitan/initialiser.py]: Can you break this into 2 statements?

Comment (uberspot) [reply kapitan/initialiser.py]: done

File: kapitan/initialiser.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    Args:
+        directory (string): path which to initialise, directory is assumed to exist
+    """
+
+    current_pwd = os.path.dirname(__file__)
+    templates_directory = os.path.join(current_pwd, 'inputs', 'templates')
+
+    copy_tree(templates_directory, directory)
+
+    # create lib/kube.libjsonnet
```
Comment (ramaro) [inline kapitan/initialiser.py]: Not sure we want to use kube.libjsonnet here, it makes it too tied to K8s

Comment (uberspot) [reply kapitan/initialiser.py]: removed.

File: kapitan/inputs/templates/components/my_component/my_component.jsonnet [hunk 5]
```diff
@@ -0,0 +1,7 @@
+local kube = import "lib/kube.libjsonnet";
```
Comment (ramaro) [inline kapitan/inputs/templates/components/my_component/my_component.jsonnet]: remove kube.libjsonnet here as well.

**Pull request merged**